packages/shared/src/hooks/log/useLogQueue.ts

```diff
@@ -79,6 +80,12 @@ export default function useLogQueue({
           const blob = new Blob([JSON.stringify({ events })], {
             type: 'application/json',
           });
+          if (isDevelopment && window) {
+            window.postMessage({
+              type: 'FYLLA_LOG_DO_NOT_USE_OR_YOU_WILL_BE_FIRED',
+              events,
+            });
+          }
```

Comment:
Wouldn't it be better to actually send this in logEvent itself, then it becomes agnostic to direct sends or beacons?

Reply:
Direct sends are intercepted directly by the extension.
But we can omit the network interception, and only use the `postMessage` for both.

Reply:
Non blocking but we should use https://developer.chrome.com/docs/extensions/reference/api/runtime#method it is more secure and specific only to extension. This can be listened by any 3rd party script.